src/app/components/ImageCard/ImageCard.tsx

```diff
@@ -16,11 +19,16 @@ export const ImageCard = ({
   children,
   imageFirst = true,
   imageStyle = {},
+  cardBackground = 'white',
+  xlHideImage = false,
 }: ImageCardProps) => (
-  <Container imageFirst={imageFirst}>
+  <Container imageFirst={imageFirst} cardBackground={cardBackground}>
     <ImageContainer imageFirst={imageFirst}>
       <Image
-        className="h-full w-full object-cover"
+        className={classNames(
```

Comment:
Is there any way we could push this image to the left rather than pushing the text to the right? The text in the design seems to line up with the image above it but currently doesn't in this code due to the left padding pushing it right (`xl:pl-10` on `TextContainer`).
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/7b85839c-beee-4a7d-9a54-73138ef71d3d">

Reply:
good catch